onnxruntime/src/error.rs

```diff
@@ -184,3 +190,10 @@ pub(crate) fn status_to_result(
     let status_wrapper: OrtStatusWrapper = status.into();
     status_wrapper.into()
 }
+
+/// A wrapper around a function on OrtApi that maps the status code into [OrtApiError]
+pub(crate) unsafe fn call_ort<F: FnMut(sys::OrtApi) -> *const sys::OrtStatus>(
+    mut block: F,
+) -> std::result::Result<(), OrtApiError> {
+    status_to_result(block(g_ort()))
```

Comment:
I know naming things is hard and hate to nitpicks on this, but I don't understand the `block` name. Is it as in "code block"? Or "blocking code"? I guess simply using `f` would be fine, would it?

Reply:
Nitpick away! It's a habit I picked up from the Kotlin community, where `block` is typical for a closure that gets called in some fairly generic wrapper logic, but if it's not clear to you then it's a bad name! `f` it is.